plugins/postgres/dbt/adapters/postgres/relation_configs/materialized_view.py

```diff
@@ -66,17 +66,19 @@ def from_dict(cls, config_dict: dict) -> "PostgresMaterializedViewConfig":
         return materialized_view
 
     @classmethod
-    def from_model_node(cls, model_node: ModelNode) -> "PostgresMaterializedViewConfig":
-        materialized_view_config = cls.parse_model_node(model_node)
+    def from_relation_config(
```

Comment:
Based on other comments, I would name this `from_config`. Also, update the return type to `Self`. It not only makes `mypy` happy, it means we can push this method up to the base class. The only reason it's at this level is to update the return type.